content/en/logs/workspaces/sql_reference.md

```diff
@@ -0,0 +1,57 @@
+---
+title: SQL Reference
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Overview
+
+SQL in Analysis cells allows you to analyze and manipulate data within Log Workspaces. This documentation covers the SQL support available in Log Workspaces and includes:
+- [SQL functions](#functions)
+- [Syntax compatible with PostgreSQL](#syntax)
+
+
+{{< img src="/logs/workspace/sql_reference/sql_syntax_analysis_cell.png" alt="Example workspace cell with SQL syntax" style="width:100%;" >}}
+
+## Functions
+
+The following SQL functions are supported:
+
+| Function         | Description                                                        |
+|------------------|--------------------------------------------------------------------|
+| `MIN`            | Returns the minimum value across all input values                  |
+| `MAX`            | Returns the maximum value across all input values                  |
+| `COUNT`          | Returns the number of input values that are not null               |
+| `SUM`            | Returns the summation across all input values                      |
+| `AVG`            | Returns the average value (arithmetic mean) across all input values|
+| `CEIL / CEILING` | Returns the value rounded up to the nearest integer                |
+| `FLOOR`          | Returns the value rounded down to the nearest integer              |
+| `ROUND`          | Returns the value rounded to the nearest integer                   |
+| `LOWER`          | Returns the string as lower case                                   |
+| `UPPER`          | Returns the string as uppercase                                    |
+| `ABS`            | Returns the absolute value                                         |
+| `COALESCE`       | Returns the first non-null value                                   |
+
+## Syntax
```

Comment:
Note that the heading above that links to this specifies "Postgres-compatible syntax", but this does not. I think it's worth making it clear that this syntax is specifically in line with either Postgres, or "standard" SQL.

Here also, it's worth having more concrete examples of these various operators such as `LIKE` and `CASE`. I would really suggest having a few working, concrete examples here.

Reply:
```suggestion
## Syntax

The following syntax is compatible with PostgreSQL.

```